src/pathwayvis/services/actions/base.ts

```diff
@@ -47,11 +47,13 @@ export abstract class ReactionAction extends Action {
         })),
       // 'objective': this.shared.objectiveReaction,
     };
-
     // this one is needed, because the backend cannot handle null as objective atm.
     if (this.shared.objectiveReaction) {
       data['objective'] = this.shared.objectiveReaction;
     }
+    if (this.shared.objectiveDirection) {
```

Comment:
If the backend is able to handle null values, it'd be better to use consistent response data -> All keys are there all the time, they're just nulls/None-s if missing.